openstf/model/regressors/regressor.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+import numpy as np
+import pandas as pd
+from openstf.model.regressors.regressor_interface import OpenstfRegressorInterface
+
+
+class OpenstfRegressor(OpenstfRegressorInterface):
```

Comment:
Makes sense to specify the shared functionality somewhere.
Out of curiosity, was this extra class a tip from Bram?

Reply:
Yes, I pitched Bram an idea how to make it a shared function inside the interface and Bram said that it might be best to add a layer between it. To keep the interface an actual interface only.